.github/workflows/update_sdk_methods.py

```diff
@@ -424,6 +400,52 @@
+## to allow us to later action on whether they are empty or not:
+python_staging_url = ''
+go_staging_url = ''
+flutter_staging_url = ''
+
+## Check for GO SDK docs staging on local workstation:
+go_process_pid = subprocess.run(["ps -ef | grep pkgsite | grep -v grep | awk {'print $2'}"], shell=True, text=True, capture_output=True).stdout.rstrip()
+
+## If we found a staged local instance of the GO SDK docs, determine which port it is using, and build the staging URL to scrape against:
+if go_process_pid != '':
+    go_process_port = subprocess.run(["lsof -Pp " + go_process_pid + " | grep LISTEN | awk {'print $9'} | sed 's%.*:%%g'"], shell=True, text = True, capture_output=True).stdout.rstrip()
+    go_staging_url = 'http://localhost:' + go_process_port
+    if args.verbose:
+        print('DEBUG: Detected local staged Go SDK docs URL: ' + go_staging_url + '/go.viam.com/rdk')
+        print('       Using this URL for all Go content (ignoring live version).')
+
+## Check for Python and Flutter SDK docs staging on local workstation.
+## Both use http.server with a user-selected port. This command fetches all possible matches:
+http_server_process_pids = subprocess.run(["ps -ef | grep http.server | grep -v grep | awk '{print $2}'"], shell=True, text=True, capture_output=True).stdout.rstrip().split('\n')
+
+## For each http.server processes detected, make educated guesses about which is which, determine the port used, and build the staging URL to scrape against:
+for pid in http_server_process_pids:
+    http_server_pwd_result = subprocess.run(["lsof -Pp " + pid + " | grep cwd | awk {'print $9'}"], shell=True, text = True, capture_output=True).stdout.rstrip()
+    
+    ## Quality guess: Python build process always builds to this directory; safe to assume:
+    if 'docs/_build/html' in http_server_pwd_result:
+        python_process_port = subprocess.run(["lsof -Pp " + pid + " | grep LISTEN | awk {'print $9'} | sed 's%.*:%%g'"], shell=True, text = True, capture_output=True).stdout.rstrip()
+        python_staging_url = 'http://localhost:' + python_process_port
+        if args.verbose:
+            print('DEBUG: Detected local staged Python SDK docs URL: ' + python_staging_url)
+            print('       Using this URL for all Python content (ignoring live version).')
+
+    ## Mediocre guess: Flutter build process likely to have string 'flutter' in cwd, either 'viam-flutter-sdk' as cloned directly, or 'flutter' as renamed by operator.
+    ## TODO: If operators run into instances where this script misses a valid Flutter staging URL because the path to that staged HTML artifacts dir has been
+    ## renamed in a fashion that does not include the string 'flutter', then change this to just always pick up any instances of http.server that aren't already matched to 
+    ## Flutter, above. This would mean that operators cannot run an unrelated http.server instance on this workstation, which they currently can do with present config.
```

Comment:
Whoops, comment typo!

```suggestion
    ## Go, above. This would mean that operators cannot run an unrelated http.server instance on this workstation, which they currently can do with present config.
```